shell/shared/grain.js

```diff
@@ -126,13 +124,6 @@ if (Meteor.isServer) {
       promise.cancel();
     });
   });
-
-  function cleanupExpiredTokens() {
```

Comment:
Oops, I meant the previous comment as a continuation of the discussion below about deleting tokens when grains get deleted.